ext/exiv2/extconf.rb

```diff
@@ -1,4 +1,12 @@
 require 'mkmf'
+
+RbConfig::MAKEFILE_CONFIG['CC'] = ENV['CC'] if ENV['CC']
+RbConfig::MAKEFILE_CONFIG['CCFLAGS'] = ENV['CCFLAGS'] if ENV['CCFLAGS']
+RbConfig::MAKEFILE_CONFIG['CXX'] = ENV['CXX'] if ENV['CXX']
+RbConfig::MAKEFILE_CONFIG['CXXFLAGS'] = ENV['CXXFLAGS'] if ENV['CXXFLAGS']
+
+dir_config("exiv2")
+have_library("exiv2")
```

Comment:
This may fail on EY. I believe `dir_config` and `have_library` were explicitly removed to make things work on Gentoo.

https://github.com/envato/exiv2/compare/master...4ormat:master#diff-a036e98a9240de7389bd8ad3b3e38ee9

Reply:
thanks, that was a conflict.

your link doesn't show anything tho 🤔

Reply:
Bad formatting on the URL:

`https://github.com/envato/exiv2/compare/master...4ormat:master#diff-a036e98a9240de7389bd8ad3b3e38ee9`